actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -2521,7 +2521,7 @@ def hidden_field(method, options = {})
       # * Creates standard HTML attributes for the tag.
       # * <tt>:disabled</tt> - If set to true, the user will not be able to use this input.
       # * <tt>:multiple</tt> - If set to true, *in most updated browsers* the user will be allowed to select multiple files.
-      # * <tt>:include_hidden</tt> - When <tt>multiple: true</tt> and <tt>include_hidden: true</tt>, the field will be prefixed with an <tt><input type="hidden"></tt> field with an empty value to support submitting an empty collection of files.
+      # * <tt>:include_hidden</tt> - When <tt>multiple: true</tt> and <tt>include_hidden: true</tt>, the field will be prefixed with an <tt><input type="hidden"></tt> field with an empty value to support submitting an empty collection of files. However, <tt>include_hidden</tt> will default to <tt>config.active_storage.multiple_file_field_include_hidden</tt> value if you don't specify <tt>include_hidden</tt>. To prevent submitting empty collection of files pass <tt>include_hidden: false</tt>
```

Comment:
```suggestion
      # * <tt>:include_hidden</tt> - When <tt>multiple: true</tt> and <tt>include_hidden: true</tt>, the field will be prefixed with an <tt><input type="hidden"></tt> field with an empty value to support submitting an empty collection of files. Since <tt>include_hidden</tt> will default to <tt>config.active_storage.multiple_file_field_include_hidden</tt> if you don't specify <tt>include_hidden</tt>, you will need to pass <tt>include_hidden: false</tt> to prevent submitting an empty collection of files when passing <tt>multiple: true</tt>.
```

Small rewording suggestion, what do you think?

(either way, can you squash your commits into one?)

Reply:
@skipkayhil Updated the language and squashed the commits into one.